src/java/org/apache/cassandra/tools/nodetool/stats/CompactionHistoryPrinter.java

```diff
@@ -40,15 +40,13 @@ public static StatsPrinter from(String format)
             default:
                 return new DefaultPrinter();
         }
-
```

Comment:
I know it looks better but this doesnt have anything to do with the change you are doing.

Reply:
Agreed. Guessing the IDE may have done this but please remove from the commit.